client/my-sites/upgrades/domain-management/components/icann-verification/icann-verification-card.jsx

```diff
@@ -42,9 +41,12 @@ let IcannVerificationCard = React.createClass( {
 				<div className="icann-verification__explanation">
 					{ this.translate(
 						'Use this button to resend the verification email. It contains a link to verify your address. ' +
-						'To update your email address, go to {{a}}Contacts and Privacy{{/a}}.', {
+						'{{a}}Learn more{{/a}}.', {
 							components: {
-								a: <a href={ paths.domainManagementEditContactInfo( this.props.selectedSite.slug, this.props.selectedDomainName ) } />
+								a: <a
+								href="https://en.support.wordpress.com/register-domain/#email-validation-and-verification"
```

Comment:
It'd be great if you could add this URL to the [`support.js`](https://github.com/Automattic/wp-calypso/blob/master/client/lib/url/support.js) file and reference it from there. That way, if the URL ever changes, we'll need to update it in one place :)
You can see how to use this class this class [for example here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/domain-management/transfer/icann-verification.jsx#L14).